### PR TITLE
Making the Commons extension Core / Standalone Collections

### PR DESCRIPTION
**Pull request by m-mohr**
Making the Commons extension Core as optional field in the collections, see issue #392 for a description and discussions. 

Combined this with the issue to add language how to combine (standalone) collections with extensions #344.

### CODE REVIEW

File: collection-spec/collection-spec.md [hunk 1]
```diff
@@ -107,12 +112,105 @@ The following types are commonly used as `rel` types in the Link Object of a Col
...
...*move common fields in Items to the parent Collection**, the collection specification allows one to more fields that are common across all linked Items to be moved out of the respective Items and into the parent STAC Collection, from which the Items then inherit. This provides maximum flexibility to data providers, as the set of common metadata fields can vary between different types of data. For instance, Landsat and Sentinel data always have an `eo:off_nadir` value of `0`, because those satellites are always pointed downward (i.e., nadir), while satellite that can be pointed will have varying `eo:off_nadir` values. This allows the data provider to define the set of metadata that defines the collection. While some metadata fields are more likely to be part of the common set, such as or `eo:instrument` rather than `eo:cloud_cover`, it depends on how the data provider chooses to organize their data. If a metadata field is specified in the Collection properties, it will be ignored in any Item that links to that Collection. This is important because a Collection is the metadata that is common across all Item objects. If a field is variable at all, it should not be part of the Commons.
```
Comment (cholmes) [inline collection-spec/collection-spec.md]: Two minor things here

'while satellite that can be pointed' -> 'while **satellites** that can be pointed'

Last word says 'Commons', should that be 'common fields'?

File: extensions/eo/README.md [hunk 2]
```diff
@@ -14,8 +14,9 @@ natural focus, and encourage other sensors to make their own extensions. Once th
 these fields will evolve to higher level extensions. In the meantime other implementations are welcome
 to reuse the names and definitions here.
 
-It is not necessary, but recommended to use the [Commons extension](../commons/README.md)
-(see chapter "Placing common fields in Collections").
+A lot of EO data will have common metadata across many Items. It is not necessary, but recommended	
+to place common fields in [STAC Collections](../../collection-spec/collection-spec.md).
```
Comment (cholmes) [inline extensions/eo/README.md]: Perhaps link to collection-spec.md#common-fields-and-standalone-collections so it takes people directly to the relevant section?

File: item-spec/item-spec.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ The following types are commonly used as `rel` types in the Link Object of an It
 | self         | STRONGLY RECOMMENDED. _Absolute_ URL to the Item if it is available at a public URL. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
 | root         | URL to the root STAC [Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md). |
 | parent       | URL to the parent STAC [Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md). |
-| collection   | STRONGLY RECOMMENDED. URL to a [Collection](../collection-spec/README.md), which may use the [Commons extension](../extensions/commons/README.md) and holds common fields of this and other Items (see chapter '[Collections](#Collections)' for more explanations). _Absolute_ URLs should be used whenever possible. |
+| collection   | STRONGLY RECOMMENDED. URL to a [Collection](../collection-spec/README.md), which may hold common fields of this and other Items (see chapter '[Collections](#Collections)' for more explanations). _Absolute_ URLs should be used whenever possible. |
```
Comment (cholmes) [inline item-spec/item-spec.md]: I'd link the words 'common fields' directly to collection-spec.md#common-fields-and-standalone-collections so people don't have to dig a lot to figure this out.

**Comment by m-mohr** [comment]
Thanks, @cholmes. Fixed all issues.

**Pull request merged**